app.py

```diff
@@ -163,5 +163,5 @@ def log_message(self, format, *args):
 
 
 if __name__ == '__main__':
-    httpd = ProxyHTTPServer(('127.0.0.1', 8080), partial(ProxyMetricsHandler, ProxyConfig()))
+    httpd = ProxyHTTPServer(('0.0.0.0', 8080), partial(ProxyMetricsHandler, ProxyConfig()))
```

Comment:
Thx for the pull request. openshift-prometheus-proxy deliberately binds to localhost only, so that it's only reachable via OAuth Proxy. This should still work on OCP 4 as the same configuration is used by the OCP 4 monitoring stack. Have you seen issues with this code on OCP 4?